model-engine/model_engine_server/core/celery/app.py

```diff
@@ -475,6 +475,8 @@ def _get_broker_endpoint_and_transport_options(
         # backoff_policy, etc., then we can expose broker_transport_options in the top-level celery() wrapper function.
         # Going to try this with defaults first.
         out_broker_transport_options["region"] = os.environ.get("AWS_REGION", "us-west-2")
+        out_broker_transport_options["wait_time_seconds"] = 0
+        out_broker_transport_options["polling_interval"] = 5
```

Comment:
also does this add any additional overhead for async requests because we're polling more often? would the volume of async GET requests scale proportionally to this

Reply:
I think this controls how often the celery workers poll SQS. It shouldn't affect any quantity of inbound requests to the gateway.